force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,27 @@
+trigger AccountTrigger on Account (before insert, after insert) {
+    if(Trigger.isBefore && Trigger.isInsert){
+      for(Account acc : Trigger.New) {
+        //check if account is null on contact
+        if(acc.Type == null ) {
+            acc.Type = 'Prospect';
+        }
+        if(acc.ShippingStreet != null && acc.ShippingPostalCode != null && acc.ShippingCountry != null && acc.ShippingCity != null && acc.ShippingState != null) {
```

Comment:
Try to separate these if statements and make the update for each one at a time instead of checking if all of htem are not null.  Each field should be checked separately.